dev/bin/interaction/create-source.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/sh
+
+source "$(dirname "${BASH_SOURCE[0]}")/common.sh"
+source_name=$TODAY_SOURCE_NAME
+source_type='slack'
+
+usage() {
+    echo 'Usage: create-source.sh [OPTIONS]'
+    echo
+    echo 'Options:'
+    echo '  -n                  Source name. Default is the generated $TODAY_SOURCE_NAME'
+    echo '  -t                  Source type. Default is `slack`. Available values: slack, AwsS3'
```

Comment:
Ah, the sweet smell of camel case (arguably `awss3` would be easier to remember).